Tensile/GenerateSummations.py

```diff
@@ -95,36 +126,24 @@ def GenerateSummations(userArgs):
         localLogicPath = ensurePath(os.path.join(currentPath, "logic"))
         localLogicFilePath = os.path.join(localLogicPath, logicFileBaseName)
         
+        # Here we read in two version of the logic the first one fills the solutions with
+        # defaults and modifies some of the parameters. The final logic file should be the 
+        # same as the initial logic with the summation model added. To preseve the original
+        # logic we also read in the raw unaltered version of the logic and stage the content
+        # to write the final logic.
         logic = LibraryIO.readLibraryLogicForSchedule(logicFileName)
+        rawLogic = readSolutionRaw(logicFileName)
 
-        (scheduleName, deviceNames, problemType, solutionsForSchedule, \
-           indexOrder, exactLogic, rangeLogic, _, architectureName) = logic
-
-        naming = Solution.getMinNaming(solutionsForSchedule)
+        (versionStringR, scheduleNameR, architectureNameR, deviceNamesR, problemTypeStateR,\
+            solutionStatesR, indexOrderR, exactLogicR, rangeLogicR, otherFieldsR) =\
+            rawLogic
 
```

Comment:
How does this behave as 'None', when no file is found?

Reply:
Skipping if there is a read failure

Reply:
Continuing past an error in this case will lead to people believing that the script has run successfully, especially if it's run by another script.  Why not just allow the file open exception to cause the script to exit?

Reply:
I changed this to exit out.